styles.css

```diff
@@ -83,7 +83,7 @@ body {
   position: absolute;
   background-color: #3c3a39;
   width: 100%;
-  height: 100%;
+  height: 70%;
   z-index: 100;
 }
```

Comment:
[REQUIRED] - I noticed that you have recently changed the height of `.mobile_nav_container` to `70%` which has affected your design.

Please kindly adjust your styling so that when the mobile menu appears, it covers the entire height of the screen to match the template on Figma. 🛠💪

Your design
![image](https://user-images.githubusercontent.com/78251927/230998919-debb474e-b791-4935-86c7-cdfa944e9cf9.png)

Figma
![image](https://user-images.githubusercontent.com/78251927/230999399-6138d0be-8a8d-4de5-92f1-780c3e656d6c.png)

[OPTIONAL] - In other to prevent page scroll while the mobile menu is displayed, we can toggle CSS ```overflow: hidden ``` on the `<body>` tag, which will prevent the page scroll. Improving the user experience of your website. 😎🛠